src/test/java/com/example/javaOjt/integrationTests/AuthorIntegrationTest.java

```diff
@@ -97,7 +100,117 @@ void getAuthorTest_withBooks_true() throws Exception {
+    Assertions.assertEquals("太宰治", resultAuthors.getAuthorsSummaryList().getFirst().name());
+    Assertions.assertEquals(2, resultAuthors.getAuthorsSummaryList().getLast().id());
+    Assertions.assertEquals("夏目漱石",
+        resultAuthors.getAuthorsSummaryList().getLast().name());
+  }
+
+  @Test
+  void getAuthorsTest_illegalAttribute() throws Exception {
+    // Perform the GET request to retrieve authors
+    MvcResult result = mockMvc.perform(
+            MockMvcRequestBuilders.get(AUTHOR_BASE_URL).param("sort_by", "v50"))
+        .andExpect(MockMvcResultMatchers.status().isBadRequest())
+        .andExpect(MockMvcResultMatchers.content().contentType(MediaType.APPLICATION_JSON))
+        .andReturn();
+
+    // Parse the error response
+    OjtExceptionResponse resultErr = objectMapper.readValue(
+        result.getResponse().getContentAsString(),
+        OjtExceptionResponse.class);
+
+    // Assertions to verify the error response
+    Assertions.assertNotNull(resultErr);
+    Assertions.assertNotNull(resultErr.getError());
+    Assertions.assertEquals("Invalid sort_by attribute",
+        resultErr.getError().message());
+    Assertions.assertEquals(400, resultErr.getError().code());
+  }
+
+  @Test
+  void getAuthorsTest_illegalOrder() throws Exception {
+    // Perform the GET request to retrieve authors
+    MvcResult result = mockMvc.perform(
+            MockMvcRequestBuilders.get(AUTHOR_BASE_URL).param("order", "kfc"))
+        .andExpect(MockMvcResultMatchers.status().isBadRequest())
+        .andExpect(MockMvcResultMatchers.content().contentType(MediaType.APPLICATION_JSON))
+        .andReturn();
+
+    // Parse the error response
+    OjtExceptionResponse resultErr = objectMapper.readValue(
+        result.getResponse().getContentAsString(),
+        OjtExceptionResponse.class);
+
+    // Assertions to verify the error response
+    Assertions.assertNotNull(resultErr);
+    Assertions.assertNotNull(resultErr.getError());
+    Assertions.assertEquals("Invalid order",
+        resultErr.getError().message());
+    Assertions.assertEquals(400, resultErr.getError().code());
+  }
+
+  @Test
+  void getAuthorsTest_emptyList() throws Exception {
+    // Perform the GET request to retrieve authors
+    MvcResult result = mockMvc.perform(
+            MockMvcRequestBuilders.get(AUTHOR_BASE_URL).param("ids", "65535"))
+        .andExpect(MockMvcResultMatchers.status().isOk())
+        .andExpect(MockMvcResultMatchers.content().contentType(MediaType.APPLICATION_JSON))
+        .andReturn();
+
+    // Parse the error response
+    GetAuthorsResponse resultEmptyList = objectMapper.readValue(
+        result.getResponse().getContentAsString(), GetAuthorsResponse.class);
+
+    // Assertions to verify the error response
+    Assertions.assertNotNull(resultEmptyList);
+    Assertions.assertEquals(new ArrayList<AuthorInfo>(), resultEmptyList.getAuthorsSummaryList());
```

Comment:
`isEmpty()` の方が読みやすいと思います。

```suggestion
    Assertions.assertTrue(resultEmptyList.getAuthorsSummaryList().isEmpty());
```